docs/docs/guides/web3_plugin_guide/plugin_users.md

```diff
@@ -29,9 +25,7 @@ Unless otherwise mentioned by the plugin author, installing a plugin should be a
 
 The `.registerPlugin` method is what we're going to be using to add a plugin to an instance of a class sourced from web3.js' modules (i.e. `Web3` or `Web3Eth`). This method only exists on classes that extend `Web3Context`, so it may not be available on every class you import from a Web3.js package.
```

Comment:
```suggestion
The `.registerPlugin` method is what you're going to be using to add a plugin to an instance of a class sourced from web3.js' modules (i.e. `Web3` or `Web3Eth`). This method only exists on classes that extend `Web3Context`, so it may not be available on every class you import from a Web3.js package.
```